webapp/components/sidebar.jsx

```diff
@@ -149,11 +150,21 @@ export default class Sidebar extends React.Component {
 
         // close the LHS on mobile when you change channels
         if (this.state.activeId !== prevState.activeId) {
-            $('.app__body .inner-wrap').removeClass('move--right');
-            $('.app__body .sidebar--left').removeClass('move--right');
+            if (this.state.closedDirectChannel) {
+                this.onUpdate(() => {
+                    this.setState({closedDirectChannel: false});
+                });
```

Comment:
Really we should not be using jQuery here to control whether the sidebar is open or not, it should be controlled by the state/props and the render method. That's a bigger refactor than I think is worth it for this ticket, though.

You're actually not using `this.state.closedDirectChannel` to affect rendering at all though so it really shouldn't be in state. Moving it to just be a field on the component itself, such as `this.closedDirectChannel`, would make more sense. It should also get around the hack of having to use this `onUpdate` function